lib/AST/Decl.cpp

```diff
@@ -5956,6 +5956,17 @@ VarDecl *VarDecl::getPropertyWrapperProjectionVar() const {
   return getPropertyWrapperBackingPropertyInfo().projectionVar;
 }
 
+void VarDecl::visitAuxiliaryDecls(llvm::function_ref<void(VarDecl *)> visit) const {
+  if (getDeclContext()->isTypeContext())
+    return;
+
+  if (auto *backingVar = getPropertyWrapperBackingProperty())
+    visit(backingVar);
+
+  if (auto *projectionVar = getPropertyWrapperProjectionVar())
+    visit(projectionVar);
+}
```

Comment:
Should this visit the lazy backing storage as well?

Reply:
Yes! I was going to try to implement local lazy in a followup PR

Reply:
(unless somebody else gets to it first 🙂 )

Reply:
Awesome! There’s an old WIP PR for it: https://github.com/apple/swift/pull/27600 which you may find helpful.

Reply:
Ah, that one got dropped on the floor because I ended up focusing entirely on figuring out a proper way to insert synthesized decls into a `BraceStmt` post-factum. They simply get *visited* here instead, right? I am sort of ashamed of myself for missing the point, but that's awesome; thanks, Holly!

Reply:
Yes, that's right - with this approach, I've added calls to `visitAuxiliaryDecls` in name lookup, the decl checker, and SILGen, so the synthesized `VarDecl`s don't need to be added to the `BraceStmt`, but they're still visited in the appropriate places.